From 0bd3266c20539740eeafd48bc897ae2c48f82222 Mon Sep 17 00:00:00 2001
From: Mark Huang <huangjc@rock-chips.com>
Date: Tue, 1 Sep 2020 11:02:21 +0800
Subject: [PATCH] ARM: dts: rockchip: rv1126-ai-cam-ddr3-v1-spi-nand: modify
 rootfs to mtd:2

The UBIFS file system does not need misc partition,
So modify the mtd of rootfs to 2.

Change-Id: I7f74cace5963dc0b5aafc6da6ab16a1a520b0257
Signed-off-by: Mark Huang <huangjc@rock-chips.com>
---
 arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1-spi-nand.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1-spi-nand.dts b/arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1-spi-nand.dts
index 044da4b86600..f9944e0202a6 100644
--- a/arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1-spi-nand.dts
+++ b/arch/arm/boot/dts/rv1126-ai-cam-ddr3-v1-spi-nand.dts
@@ -11,7 +11,7 @@
 	compatible = "rockchip,rv1126-ai-cam-ddr3-v1-spi-nand", "rockchip,rv1126";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 ubi.mtd=3 root=ubi0:rootfs rootfstype=ubifs snd_aloop.index=7";
+		bootargs = "earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 ubi.mtd=2 root=ubi0:rootfs rootfstype=ubifs snd_aloop.index=7";
 	};
 };
 
-- 
2.35.3

